python/desc/sims/GCRCatSimInterface/README.lensing

```diff
@@ -0,0 +1,11 @@
+To generate the FITS files of the lensed hosts, you will need to run the following commands only once:
```

Comment:
I think these instructions need more details.  First off, the `load_in_data_[sne,agn]` functions assume that these functions will be run from the `python/desc/sims/GCRCatSimInterface` folder, otherwise the `pd.read_csv` calls would fail.  However, I'm not sure this folder is the best place to put those data since there is already a `data` folder in this repo.  Second, the code is written so that the generated FITS images will be written to `python/desc/sims/GCRCatSimInterface/outputs`, which similarly may not be the best place.    I would move the data files to the top level `data` folder, and make it so that the user can specify where the output images should be written.